pubsub/include/marlin/pubsub/PubSubNode.hpp

```diff
@@ -4,7 +4,7 @@
 
 #ifndef MARLIN_PUBSUB_PUBSUBNODE_HPP
 #define MARLIN_PUBSUB_PUBSUBNODE_HPP
-
+		
```

Comment:
Configure your editor to trim all trailing whitespace and to end each file with a single newline. Otherwise it causes a lot of noise in diffs.